content/manuals/engine/daemon/prometheus.md

```diff
@@ -41,15 +41,15 @@ Add the following configuration:
 
 ```json
 {
-  "metrics-addr": "127.0.0.1:9323"
+  "metrics-addr": "0.0.0.0:9323"
 }
 ```
 
 Save the file, or in the case of Docker Desktop for Mac or Docker Desktop for
 Windows, save the configuration. Restart Docker.
 
-Docker now exposes Prometheus-compatible metrics on port 9323 on the loopback
-interface.
+Docker now exposes Prometheus-compatible metrics on port 9323, using the
+wildcard address `0.0.0.0`.
```

Comment:
To address Sebastiaan's point:
How about adding it as an optional note with warning, like "instead of the loopback interface, you can use the wildcard address `0.0.0.0`, but (...)"